.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: node_js
+
+services:
+  - docker
+
 script:
+  - docker build --tag web-ui .
```

Comment:
does it matter that the docker stuff is around the `yarn run build`?  seems like they're a bit independent, assuming the `docker build` includes running the `yarn build` within the docker container.

Reply:
I think the container takes a bit to start, so the build is acting as a hacky delay. I didn't love that either, so that's why I asked about healthcheck